### PR TITLE
OMA-6 Update educandu and set up site logo

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/OMA-6

### CODE REVIEW

File: src/main.less [hunk 1]
```diff
@@ -1,11 +1,11 @@
 @import url('@educandu/educandu/styles/main.less');
 
-@global-primary-color: brown;
+@global-primary-color: @pink-6;
```
Comment (adrianaluchian) [inline src/main.less:3]: It's even more pink now. Prepare for an eye burn!

Comment (ahelmberger) [reply src/main.less:3]: ouch!

Comment (laurentiu-ilici) [reply src/main.less:3]: This is weirdly not so bad :D

**Review by laurentiu-ilici — APPROVED** [review]

File: src/bundles/site-logo.js [hunk 2]
```diff
@@ -3,26 +3,26 @@ import PropTypes from 'prop-types';
...
 
-function Logo({ readonly, size }) {
+function SiteLogo({ readonly, size }) {
   const classes = classNames({
-    'Logo': true,
-    'Logo--readonly': readonly,
-    'Logo--big': size === 'big'
+    'SiteLogo': true,
+    'SiteLogo--readonly': readonly,
+    'SiteLogo--normal': size === 'normal'
```
Comment (ahelmberger) [inline src/bundles/site-logo.js:10]: Isn't it a little bit weird, that `normal` is an override of `small`? I found it more logical to have `big` as an override of `normal`.

Comment (adrianaluchian) [reply src/bundles/site-logo.js]: Actually this is a leftover from the old `site-logo`. I made it better now, please check.

File: src/main.less (hunk 1 above)
Comment (adrianaluchian) [reply src/main.less:3]: We can also change the colors, I don't care much either way so far, just wanted to make the links visible. And make sure we don't forget what website we're on when we run OMA 😋

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**